test/audiovideocontroller/DefaultAudioVideoController.test.ts

```diff
@@ -825,12 +826,15 @@ describe('DefaultAudioVideoController', () => {
       await delay(defaultDelay);
       expect(sessionStarted).to.be.true;
       expect(sessionConnecting).to.be.true;
-      audioVideoController.setVideoMaxBandwidthKbps(100);
       audioVideoController.handleHasBandwidthPriority(false);
       audioVideoController.handleHasBandwidthPriority(true);
       audioVideoController.handleHasBandwidthPriority(true);
```

Comment:
This is not related to this PR. But I'm curious why we need two of this?

Reply:
Looking at the `expect` lines. I don't think calling `handleHasBandwidthPriority` multiple times is not required. It might be for idempotence check.